schema.sql

```diff
@@ -33,4 +33,25 @@ ALTER TABLE animals
     ADD species_id INT NULL REFERENCES species(id) ON DELETE CASCADE,
     ADD owner_id INT NULL REFERENCES owners(id) ON DELETE CASCADE;
 COMMIT;
-SELECT * FROM animals;
\ No newline at end of file
+SELECT * FROM animals;
+
+/*day-4*/
+CREATE TABLE vets (
+    id INT primary key generated always as identity,
+    name VARCHAR(80) NOT NULL,
+    age INT NOT NULL,
+	date_of_graduation DATE
+);
+
+CREATE TABLE specializations (
+    id INT primary key generated always as identity,
+    vets_id INT NULL REFERENCES vets(id),
+	species_id INT NULL REFERENCES species(id)
+);
+
+CREATE TABLE visits (
+    id INT primary key generated always as identity,
+    vets_id INT NULL REFERENCES vets(id),
+	animals_id INT NULL REFERENCES animals(id),
+	visit_date DATE NULL
```

Comment:
- [ ] [OPTIONAL]  I think It will be better if you remove the NULL constraint, as currently the vets_id, animals_id and visit_date columns are defined as NULL-able, which means that a Visits record can exist without an associated vet or animal or date of visit.